compiler/rustc_errors/src/lib.rs

```diff
@@ -996,14 +996,14 @@ impl Handler {
         self.inner.borrow_mut().span_bug(span, msg)
     }
 
-    /// For documentation on this, see `Session::delay_span_bug`.
+    /// For documentation on this, see `Session::span_delayed_bug`.
```

Comment:
Can you amend this commit with a note that this used to be called `delay_span_bug`? People will probably not like that this is renamed 😓

Reply:
:+1: